sonar-project.properties

```diff
@@ -0,0 +1,18 @@
+sonar.projectKey=metamask-extension
+sonar.organization=consensys
+
+# This is the name and version displayed in the SonarCloud UI.
+sonar.projectName=MetaMask Extension
+#sonar.projectVersion=1.0
+
+# Root for sonar analysis.
+sonar.sources=app/
+
+# Excluded project files from analysis.
+#sonar.exclusions=
+
+# Inclusions for test files.
+sonar.test.inclusions=**.test.**
```

Comment:
shall we also include e2e tests `xxx.spec.js`

Reply:
Good question. I don't think so because Sonar considers unit tests only for coverage.